ddmark/README.md

```diff
@@ -30,7 +30,7 @@ It also allows us to define custom rules to apply to our structures, and focus t
 - ExclusiveFields:
   - `// +ddmark:validation:ExclusiveFields={<fieldName1>,<fieldName2>,...}`
   - Applies to: `<struct>` type
-  - Asserts only one of the given fieldnames isn’t `nil`
+  - Asserts that if `<fieldname1>` can only be non-nil iff all of the other fields are `nil`
```

Comment:
```suggestion
  - Asserts that if `<fieldname1>` can only be non-nil if all of the other fields are `nil`
```

Reply:
Intentional! Meant "if and only if" here. Is that shorthand unfamiliar? I can remove it then

Reply:
Sugg: - Asserts that `<fieldname1>` can only be non-nil if all of the other fields are `nil`

Alternative (seems more readable to me): - Asserts that `<fieldname1>` can be non-nil only if all of the other fields are `nil`
(that's one too many ifs?

Reply:
TIL `iff` :D sorry